group-actors/src/main/java/org/sunbird/actors/CreateGroupActor.java

```diff
@@ -62,17 +61,20 @@ private void createGroup(Request actorMessage) throws BaseException {
     // adding members to group, if members are provided in request
     List<Map<String, Object>> reqMemberList =
         (List<Map<String, Object>>) actorMessage.getRequest().get(JsonKey.MEMBERS);
-    logger.info("Adding members to the group: {} started", group.getName());
+    logger.info("Adding members to the group: {} groupId started", group.getName());
```

Comment:
Group name is not a unique thing. We could capture the name just while creating. Rest could log only group id.

Reply:
ok done,